lib/json_matchers/matcher.rb

```diff
@@ -31,7 +31,9 @@ def validator
     def build_and_populate_document_store
       document_store = JsonSchema::DocumentStore.new
 
-      Dir.glob("#{JsonMatchers.schema_root}/**/*.json").
+      # follow one symlink and direct children
+      # http://stackoverflow.com/questions/357754/can-i-traverse-symlinked-directories-in-ruby-with-a-glob
+      Dir.glob("#{JsonMatchers.schema_root}/**{,/*/**}/*.json").
```

Comment:
Thanks for this contribution!

In my experience, I've found that code comments with URLs like this can be really helpful to keep close to the code that might strike the next developer as an oddity.

With that in mind, how do you feel about extracting out a private method named in a way that communicates the intent behind this decision?

One benefit is that a well-named private method is more likely to survived future refactorings. Personally, I feel that it also serves as an anchor for code comments like this.